docs/en/ingest-management/elastic-agent/configuration/elastic-agent-kubernetes-autodiscovery.asciidoc

```diff
@@ -2,7 +2,7 @@
 = Kubernetes autodiscovery with {agent}
 
 [discrete]
-== Autodiscover targeted Pods based on conditions
+== Conditions based autodiscover
```

Comment:
Are the changes made here intended to align with Filebeat? Or some industry terminology? Just curious because I don't see these words being more valuable as search terms.

Reply:
It was suggested by @gizas and @mlunadia . I guess for readability reasons.